test_psa_target.py

```diff
@@ -152,6 +152,22 @@ def _erase_flash_storage(args, suite):
             "-Intel",
         ]
 
+    if args.mcu == "ARM_MUSCA_S1":
```

Comment:
Please add regex test logs for auto comparison.

Reply:
Logs are added.